packages/meta-extraction/src/index.js

```diff
@@ -52,11 +52,17 @@ const extractAbstract = async (url, skipTidy = false) => {
     }
     // status filter
     const status = response.status().toString();
+    const responseUrl = response.url();
     console.log(`HTTP status code: ${status}`);
 
-    if (shouldEnableMultiRedirect(response.url())){
+    if (shouldEnableMultiRedirect(responseUrl)){
       await browserInstance.close();
-      return extractAbstract(response.url(), true);
+      if (status.startsWith('3') || status.startsWith('2')) {
+        console.log(`Redirecting to ${responseUrl}`);
```

Comment:
Do we need this console.log?

Reply:
all the console.log won't appear in process function log
they may be used when the tests are executed in github actions